volume/rsync/rsync.go

```diff
@@ -649,7 +649,10 @@ func (v *RsyncVolume) Rollback(label string) (err error) {
 }
 
 // Export implements volume.Volume.Export
-func (v *RsyncVolume) Export(label, parent string, writer io.Writer) error {
+func (v *RsyncVolume) Export(label, parent string, writer io.Writer, excludes []string) error {
+	if len(excludes) > 0 {
+		glog.Warning("rsync backups do not support excluding directories")
```

Comment:
Technically true, but I don't want to dump effort into the rsync driver since nothing uses it anymore.